sotodlib/preprocess/preprocess_util.py

```diff
@@ -385,12 +394,13 @@ def load_and_preprocess(obs_id, configs, context=None, dets=None, meta=None,
         pipe = Pipeline(configs["process_pipe"], logger=logger)
         aman = context.get_obs(meta, no_signal=no_signal)
         pipe.run(aman, aman.preprocess, select=False)
-        return aman
+        return aman, full_aman
```

Comment:
Did you look into if making this function now return two outputs will break anyone's existing workflows? And same question applies for `multilayer_load_and_preprocess`